### ISSUE TITLE
Remove --open call on npm start

### ISSUE THREAD

**Issue by sthzg** [react-webpack-generators/react-webpack-template#46]
As brought up in [generator-react-webpack #266](https://github.com/react-webpack-generators/generator-react-webpack/issues/266#issuecomment-239012817) 

- [ ] remove the ` --open` call on `npm start` 
- [ ] log dev server address to console after the inital webpack build is ready

@weblogixx should I tackle this?

**Comment by weblogixx** [comment]
Hi @sthzg, sure just go ahead.

**Comment by sthzg** [comment]
@weblogixx I've split this issue out in two:

#46 => V1 
#47  => V2.

In the V2 version `npm start` directly invokes the `webpack-dev-server` cli, so there is no way that I know of to listen for the end of the (first) compilation to print a message to the console afterwards. Webpack however prints it by default on the very top (which after bundling is in the scrollback). 

If this isn't good enough I don't see another way than bootstrapping the `webpack-dev-server` in a script file so that we can register a listener to the `done` event of `compile`.

**Comment by weblogixx** [comment]
Yes, there seems to be no way to do this with the command line currently :(. Have to think about that.

**Comment by sthzg** [comment]
Would there be something speaking against providing the devserver script in a `devserver.js` script (like it has been in V3 in `server.js`)? After all, it seems to be just a small wrapper around the CLI, or am I missing something?

**Comment by weblogixx** [comment]
The only reason I removed it was that there is one file less and you can parse cli args into the launcher directly.
